src/scss/components/_button.scss

```diff
@@ -4,11 +4,13 @@
   border-radius: 6px;
   color: $color-white;
   display: inline-block;
+  font-family: $font-family-sans;
```

Comment:
> One issue I see now is that the Submit buttons on the forms look kinda funny on iOS since I added that font to the button class. It's likely that was already an issue, but I hadn't noticed it until now. Maybe it's more a browser default setting? It really doesn't bother me at all but if you or Melanie want me to work on it I'm happy to.

Try adding this to `.button`:

```
-webkit-appearance: none;
appearance: none;
```

That'll wipe iOS' default button styles in favor of all of the ones you're adding. Once I do that, the buttons look correct again.

https://css-tricks.com/almanac/properties/a/appearance/

Reply:
> Also it looks like the inputs for date and time on the Booking page don't stretch the whole width of the page on iOS. Tried messing with it a bit but couldn't fix it. I'll try some more.

I think adding these appearance properties to `.page-form__form-input` would fix this too. It looks like iOS is applying default styles to those inputs (to the text ones too, it's just less noticeable) and that would undo them. You'd still get iOS' date and time pickers once you tapped them, so I think you might as well add the properties.

Reply:
Totally fixed both the buttons and the form elements. Awesome! Dang I needed to get better at researching solutions like that.